README.md

```diff
@@ -114,7 +114,99 @@ and configures the jongo/jackson mapper with the `DefaultScalaModule`. To use th
 Usage
 -----
 
-*TODO: needs to be updated for play-jongo 2+*
+**Play Framework 2.5.x**
+
+A way to use PlayJongo is to create a repositories package containing repository classes, one for each model. A repository class contains the PlayJongo injection, the collection to use and all methods to access to the collection members.
+The package structure should be similar to the following:
+```
+|- controllers
+|- models
+|- repositories
+```
+
+Model example:
+
+```java
+public class Users  {
```

Comment:
Of course naming is up to you, but it could also be singular / "User" here. WDYT?